frontend/src/metabase/dashboard/containers/DashboardHeader.styled.tsx

```diff
@@ -26,9 +31,18 @@ export const DashboardHeaderButton = styled(Button)<{ isActive?: boolean }>`
   svg {
     vertical-align: middle;
   }
+
+  ${breakpointMaxSmall} {
+    ${props =>
+      css`
+        ${props.visible ? "" : "display: none;"}
+      `}
```

Comment:
We lose [hinting](https://github.com/metabase/metabase/assets/6830683/b51bf324-33f1-40f9-a778-ba538bb2e334) if we define CSS in simple strings.


```suggestion
    ${props =>
      !props.visible &&
      css`
        display: none;
      `}
```